.gitignore

```diff
@@ -6,6 +6,7 @@
 *$
 *.bak
 *.diff
+.idea
```

Comment:
The [recommendation](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore) seems to be `.idea/`.